Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Notification/DeleteNotification.cs

```diff
@@ -37,9 +38,7 @@ public static class DeleteNotification
             
             log.LogInformation($"Got reference to {Constants.COLLECTION_NOTIFICATION} collection on {Constants.DATABASE_NOTIFY_MTA} database");
```

Comment:
This log can be removed, and forgot the collection above this log.

Reply:
Fixed